src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="big-form"
+      method="get"
+      action="#">
+      <img
+        class="big-image"
+        src="./images/Search.svg"
+        alt="search logo">
+      <label>
+        <input
+          class="input big-input"
+          type="text"
+          name="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="big"
+        >
+      </label>
+    </form>
+    <form
+      class="small-form"
+      action="#"
+      method="get"
+    >
+      <img
+        class="small-image"
+        src="./images/Search.svg"
+        alt="search logo"
+      >
```

Comment:
Do not insert such images into html, use css for setting background:
https://codepen.io/adskiy/pen/mdmLqXB
example